src/cosmology/compat/camb/constants.py

```diff
@@ -5,9 +5,11 @@
 From the :mod:`cosmology.api`, the list of required constants is:
 
 - G: Gravitational constant G in pc km2 s-2 Msol-1.
+- speed_of_light: Speed of light in km s-1.
 """
 
 __all__ = ["G"]
 
 
 G = 4.30091727003628e-3  # [pc km2 s-2 Msol-1]  CODATA 2018 value
+speed_of_light = 299792.458  # [km s-1]  CODATA 2018 value
```

Comment:
(NB: If we are using single-letter constants like `G`, well then there is no clearer single-letter constant than `c`.)

Reply:
I'll make the change globally. `c` suits me fine as well.

Reply:
r.e. G. would it be Bette to switch to the SI definition, like CAMB?

Reply:
It's better constrained than the mass of the sun, which we would need to carry around as another constant.

Reply:
I think G in cosmological units covers so much of the use cases, we can keep it as it is for the time being.  I doubt we will ever run into an inconsistency because of different definitions of Msun, because every function where that could happen will receive or produce masses in Msun anyway.